src/cfViewCommands.ts

```diff
@@ -369,9 +369,19 @@ export async function cmdGetUpsServiceInstances(options?: UpsServiceQueryOprions
 export async function cmdGetServiceInstances(serviceQueryOptions?: ServiceQueryOptions, progressTitle?: string): Promise<ServiceInstanceInfo[]> {
     let query: IServiceQuery;
     if (serviceQueryOptions) {
+        //prepare a query to filter the service list by service plan (e.g.: hdi-shared)
```

Comment:
consider to move the init of serviceInfo closer to usage

Reply:
the serviceInfo and service plans are parameters that we are using for the composePlansGuidListForQuery
and I prepare it before it. 
so I don't see to a reason to change it

Reply:
done